app/Resources/views/article.html.twig

```diff
@@ -316,6 +316,35 @@
 
     {%- endif %}
 
+    <script>
+        {
+            "@context": "https://schema.org",
+            "@type": "Report",
+            "mainEntityOfPage": {
+                "@type": "WebPage",
+                "@id": "{{ url('article', {id: item.id}) }}"
+            },
+            "headline": "{{ item.fullTitle|striptags }}",
+            "image": [
+                "https://iiif.elifesciences.org/journal-cms/cover%2F2019-07%2F46846-a_striking_image.jpg/0,179,1800,543/1114,336/0/default.jpg"
+            ],
+            "datePublished": "{{ item.publishedDate|date('Y/m/d', 'UTC') }}",
+            "author": [{% for author in item.authors %}{
+                "@type": "Person",
+                "name": "{{ author.toString }}"
+            }{%- if not loop.last %}, {% endif -%}{% endfor %}],
```

Comment:
Not all authors are people. Suggest we just use a list of plain strings if we're not adding any other details.